config/config.go

```diff
@@ -39,6 +39,7 @@ type Configuration struct {
 	}
 	PassStrength      int    `default:"10"`
 	UploadedImagesDir string `default:"data/images"`
+	PluginsDir        string `default:""`
```

Comment:
Could you change the default to `data/plugins`

Reply:
Okay, I set it to empty at first because I want plugin functionality to be disabled by default, now it seems that an empty directory works as well.